js/ngl/writer.js

```diff
@@ -85,7 +85,7 @@ NGL.PdbWriter = function( structure, params ){
                 // Alignment of one-letter atom name such as C starts at column 14,
                 // while two-letter atom name such as FE starts at column 13.
                 var atomname = a.atomname;
-                if( atomname.length === 1 ) atomname = " " + atomname;
+                atomname = " " + atomname;
```

Comment:
What happens with atomnames of 4 characters? The field is then 5 characters long but only 4 are allowed. Also, the comment does not match the code.